.gitmodules

```
@@ -3,4 +3,5 @@
 	url = https://github.com/smartdevicelink/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/LuxoftSDL/rpc_spec.git
+	branch = feature/0257_vehicle_data_handsOffSteering
```

```suggestion
	url = https://github.com/smartdevicelink/rpc_spec.git
```
Reminder to revert this bit before merge

---

@jacobkeeler reverted in https://github.com/smartdevicelink/sdl_core/pull/3394/commits/9fe091622f5919005c4961995a8a4f6960008c52